Example/SwiftUIExample/MobilePaymentsSwiftUIExample.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,15 @@
+{
+  "originHash" : "9ba5bf9b6f630a6dffe060602219b27b9ee75c935dd982c73df381742de27451",
+  "pins" : [
+    {
+      "identity" : "mobile-payments-sdk-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/square/mobile-payments-sdk-ios",
```

Comment:
Instead of using a remote package, you could use the local package file. That way it's always using what ever this branch is on and we don't need to remember to update it.

Reply:
Didn't even realize you could do this. Updated 🙏